wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsDFUClient.java

```diff
@@ -883,6 +887,88 @@ public int hashCode()
         return result;
     }
 
+    /**
+     * getFileAccessBlob
+     *
+     * @param accesstype
+     *            - the file access level to request to request
+     * @param filename
+     *            - the name of the target file to be accessed
+     * @param clustername
+     *            - the name of the target file's HPCC cluster (can be empty)
+     * @param expiryseconds
+     *            - the number of seconds file access is granted
+     * @param jobid
+     *            - unique identifier for access token
+     * @return - Access artifact to be propagated as part of DAFILESERV file access requests
+     * @throws Exception
+     */
+    public String getFileAccessBlob(SecAccessType accesstype, String filename, String clustername, int expiryseconds, String jobid) throws Exception
+    {
+        DFUFileAccessResponse resp = getFileAccess(accesstype, filename, clustername, expiryseconds, jobid, false, false, false);
+        if (resp != null)
+            return resp.getMetaInfoBlob();
+        return "";
```

Comment:
@rpastrana Is the resp == null case recoverable? IE does returning an empty "blob" make sense? Maybe this should be an exception or return null?